src/item.cpp

```diff
@@ -1523,6 +1523,11 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
                 }
             }
         }
+        
+        if( is_gun() && has_flag( "FIRE_TWOHAND" ) ) {
+            info.push_back( iteminfo( "DESCRIPTION",
+                                      _( "* This weapon needs <info>two free hands</info> to fire." ) ) );
```

Comment:
Definitely not for the repeating crossbow, every time you pump it, it fires
a quarrel, it's not "pump, then aim and fire", it's "aim and pump to fire".
In theory you could reload the heavy crossbow with one hand, but it would
be extremely awkward and take a very long time to do, if you want to figure
out a way to support that go ahead, but it seems exceedingly niche to me.